src/core/analysis-cache.ts

```diff
@@ -75,49 +75,78 @@ export class AnalysisCache {
     //     Could end up saving some work in the editor case.
     //     On the other hand, copying a half dozen maps with maybe 200 entries
     //     each should be pretty cheap, maybe not worth the effort.
-    const newCache = new AnalysisCache(
-        this, this.dependencyGraph.invalidatePaths(documentPaths));
+    // const newCache = new AnalysisCache(
+    //   this, this.dependencyGraph.invalidatePaths(documentPaths));
+    const dependencyGraph = this.dependencyGraph.invalidatePaths(documentPaths);
```

Comment:
Move this to near the new cache constructor below, it'll avoid confusion with like 94 below where you get dependants based on the previous graph.